docs/ProgrammingCryptol/highAssurance/HighAssurance.tex

```diff
@@ -174,6 +174,51 @@ \subsection{Capturing test vectors}
   sure the tweak to the above example (removal of dummy single formal
   parameter) works.}
 
+%~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+\subsection{Generating test vectors}
+\label{sec:gentestvec}
+
+Cryptol has the ability to generate test vectors for simple function types using the \texttt{:dumptests} REPL command.
+When \texttt{:dumptests} is given a file to dump the output and a function in scope, it will randomly generate inputs for the function and run them for you.
+The output will contain a tab delimited file in which the first column contains the output, and each column after contains the inputs to the function.
+The command \texttt{:dumptests} uses the same settings as \texttt{:check} \ref{sec:quickcheck}, this means you can set the number of tests you want using the \texttt{:set tests=100} command.
```

Comment:
suggestion: I misinterpreted this as saying that dumptests is like check except it tells you what inputs it used. That's not really the case, though, is it? Because (a) dumptests isn't specifically for properties, it's for any function, and thus (b) dumptests doesn't fail or say anything if a function returns false (as in my example in another comment).

I would suggest removing the reference to check, or having it come second in the sentence so that the emphasis is on the fact that you can change the number of tests you get:
> The size of the output of `dumptests` can be modified by using `:set tests=1000` to change the number of tests to 1000. This is the same configuration setting that is used to determine the number of tests run for the `:check` command, as described in Section [...].

Reply:
It also looks like if you change the base `:set base=2` you'll get the test output in that format. Is this something users might care about?

Reply:
Right, this is why I wanted to reference all of the things in `:check` as it is essentially using the random generation from there... but I agree that it is a bit confusing as is.

Reply:
Reworded but did keep the reference to quick check in there as it appears that this will use generators you create as well which is good to know.